USAGE_desktop_Vulkan.md

```diff
@@ -261,7 +261,9 @@ option values.
-| Hotkey Capture Trigger Frames                  | GFXRECON_CAPTURE_TRIGGER_FRAMES                         | STRING  | Specify a limit on the number of frames to be captured via hotkey.  Example: `1` will capture exactly one frame when the trigger key is pressed. Default is: Empty string (no limit)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |
+| Hotkey Capture Trigger Frames                  | GFXRECON_CAPTURE_TRIGGER_FRAMES                         | STRING  | Specify a limit on the number of frames to be captured via hotkey.  Example: `1` will capture exactly one frame when the trigger key is pressed. Default is: Empty string (no limit)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
+| Use asset file                                 | GFXRECON_CAPTURE_USE_ASSET_FILE                         | BOOL    | When set to `true` assets (images, buffers and descriptors) will be stored separately into an asset file instead of the capture file.                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
```

Comment:
wouldn't desktop also benefit from a dump asset file trigger like android has?

Reply:
I'm pretty sure that doesn't make sense, for the same reason `capture_android_trigger` doesn't exist on desktop - you can't modify environment variables at runtime (but android properties can be modified at runtime). Though a hotkey on desktop would make more sense.